src/commands/general/star.ts

```diff
@@ -76,9 +76,9 @@ const star: CommandInt = {
       attachment.push(new MessageAttachment("./img/star.png", "star.png"));
 
       // Send an embed message with the star to the user.
-      await userToStarMentioned.send(
+      await channel.send(
         new MessageEmbed()
-          .setTitle("You got a gold star!")
+          .setTitle(`Congratulations, ${userToStarMentioned.toString()}, you got a gold star!`)
```

Comment:
```suggestion
          .setTitle(
            `Congratulations, ${userToStarMentioned.username}, you got a gold star!`
          )
```
Tweak this to display the name correctly.